package.json

```diff
@@ -1,8 +1,13 @@
 {
-  "name": "rudolf",
-  "version": "0.1.0",
+  "name": "truth-tree",
```

Comment:
This is absolutely a bikeshed, but maybe it would be better to call it carnap-truth-tree or similar?

Reply:
Maybe. I thought I'd grab the simplelr id on npm since it was available.
There are pros and cons for either name.
If we were to make the truth tree widget usable independently of Carnap in the future, I figured it would be better to have a purely descriptive name.
Since it's already published, we can stick with the current name for now, until we decide how to manage the relationship to the parent project, long-term.